src/_pytest/fixtures.py

```diff
@@ -960,7 +965,8 @@ class FixtureFunctionMarker:
     scope = attr.ib()
     params = attr.ib(converter=attr.converters.optional(tuple))
     autouse = attr.ib(default=False)
-    ids = attr.ib(default=None, converter=_ensure_immutable_ids)
+    # Ignore type because of https://github.com/python/mypy/issues/6172.
```

Comment:
hmmm weird, that issue seems to indicate named functions should work, oh well :/